web-ui/src/components/action_item/ActionItemsPanel.jsx

```diff
@@ -27,7 +27,7 @@ const doUpdate = async (actionItem) => {
 
 const updateItem = debounce(doUpdate, 1500);
 
-const ActionItemsPanel = ({ checkinId, memberName }) => {
+const ActionItemsPanel = ({ checkinId, memberName, mockActionItems }) => {
```

Comment:
not gonna lie..I don't like this one bit.

Reply:
That was an attempt at making the test work, which I still didn't end up fixing, so I can remove it